generators/client/templates/angular/src/main/webapp/app/admin/admin.route.ts.ejs

```diff
@@ -52,17 +49,28 @@ const ADMIN_ROUTES = [
     <%_ if (websocket === 'spring-websocket') { _%>
     trackerRoute,
     <%_ } _%>
-    <%_ if (!skipUserManagement) { _%>
-    ...userMgmtRoute,
-    <%_ } _%>
     metricsRoute
 ];
 
-export const adminState: Routes = [{
+export const adminState: Routes = [
+<%_ if (!skipUserManagement) { _%>
+  {
+    path: 'user-management',
+    data: {
+      authorities: ['ROLE_ADMIN']
+    },
+    canActivate: [UserRouteAccessService],
+    loadChildren: () => import('./user-management/user-management.module').then(m => m.UserManagementModule)
+  },
+<%_ } _%>
+  {
     path: '',
     data: {
-        authorities: ['ROLE_ADMIN']
+      authorities: ['ROLE_ADMIN']
     },
     canActivate: [UserRouteAccessService],
     children: ADMIN_ROUTES
-}];
+  }
+];
+
+export const AdminRouting = RouterModule.forChild(adminState);
```

Comment:
we should either specify `RouterModule.forChild` directly in the `admin.module` or restructure this class to a routing module and rename as  `admin-routing.module`encapsulating admin routes and import that module in the `admin.module` (in the long term, I prefer this approach).